modules/ob-securely-referencing-string-parameters.adoc

```diff
@@ -0,0 +1,75 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-strategies.adoc
+
+:_content-type: CONCEPT
+[id="ob-securely-referencing-string-parameters_{context}"]
+= Securely referencing string parameters
+
+You can reference string parameters by using the `$(params.PARAM_NAME)` parameter in your build strategy steps. You can use string parameters when defining environment variables, arguments, or images in a `BuildStrategy` or `ClusterBuildStrategy` CR. 
```

Comment:
Is $(params.PARAM_NAME) a parameter or is it a syntax? Could it be referenced as `PARAM_NAME` ? I don't see any parameters reference that follow this format in our docs. There are a few other instances of this, and I'm not sure if I understand them correctly.

Reply:
This is a synatx a user must use to define a string or an array parameter in a build startegy.

Reply:
@apoorvajagtap @jkhelil Can we change the parameter syntax from $(params.PARAM_NAME) to only $(PARAM_NAME)?

Reply:
I don't think we should reference it as just `PARAM_NAME`. Any parameter to be specified in buildStrategy has to be as per `params.PARAM_NAME` syntax. For reference please check [here](https://github.com/shipwright-io/build/blob/main/samples/buildstrategy/buildah/buildstrategy_buildah_shipwright_managed_push_cr.yaml#L169).
If required, we can change the way we highlight `PARAM_NAME`, but we should keep `params.`.